open-models/serving/cloud_run_ollama_gemma2_rag_qa.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<ol><li>Typo: "VertexAI" should be "Vertex AI"</li><li>Consider rephrasing the following sentence from BEFORE to AFTER:</li></ol><p> </p><p>BEFORE:</p><p>"We will ingest a <a href="https://cloud.google.com/run/docs/overview/what-is-cloud-run" target="_blank">Cloud Run documentation page</a> to power grounded answers on the top of it."</p><p> </p><p>AFTER:</p><p>"Through RAG, we will ask Gemma 2 to answer questions about the <a href="https://cloud.google.com/run/docs/overview/what-is-cloud-run" target="_blank">Cloud Run documentation page</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Done